access_group_cache/main.py

```diff
@@ -150,6 +189,11 @@ def index():
     )
 
     for group, group_members in zip(groups, all_group_members):
+        if isinstance(group_members, Exception):
+            logging.warning(
```

Comment:
I think this should be an error. Can we please also set up alerting for this?
(Vivian said it's fairly easy to scan the logs and emit a Slack message when it triggers.)

Reply:
Yep agreed, I've made it an error and will set-up the log scanner.